### PR TITLE
Issue 10

### PR DESCRIPTION
**Pull request by SooDevv**
원문 번역, 코드 파일(ipynb) 번역 완료하였습니다.
오탈자나 번역이 매끄럽지 못한 부분은 comment 달아주시면 감사하겠습니다!

### CODE REVIEW

File: 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+# 딥러닝의 Hello World, Fashion-MNIST <br> (Fashion-MNIST with tf.Keras)
+
+[Fashion-MNIST with tf.Keras 원문 바로 가기](https://medium.com/tensorflow/hello-deep-learning-fashion-mnist-with-keras-50fcff8cd74a)
+
+> 이 문서는 MNIST 데이터 셋을 대신해서 사용할 수 있는 Fashion-MNIST 데이터 셋을 사용한 튜토리얼 입니다. <br>딥러닝 입문자들은 tf.keras 를 사용하여 손쉽게 모델을 구현할 수 있습니다.
+
+* 케라스
+* Fashoin-MNIST
```
Comment (fuzzythecat) [inline 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md]: Fashoin -> Fashion

File: 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+* Convolution Neural Networks
+
+## Fashion-MNIST with tf.Keras
+
+이 문서는 컨볼루션 신경망(CNN)을 사용하여 tf.keras로 Fashion-MNIST 데이터 셋을 분류하는 튜토리얼입니다. 몇 줄의 코드를 통해, 많은 최적화 과정 없이도 90% 이상의 정확도로 이미지를 분류 할 수있는 모델을 정의하고 학습 할 수 있습니다.
+
+![](../media/10_1.png)
+
+[Fashion-MNIST](https://github.com/zalandoresearch/fashion-mnist)는 기존의 [MNIST](http://yann.lecun.com/exdb/mnist/) 데이터 셋(10개 카테고리의 손으로 쓴 숫자)을 대신해 사용할 수 있는데, 그 이유는 동일한 이미지 크기(28x28)이며 동일한 학습 셋 (60,000), 테스트 셋(10,000)으로 나눌 수 있기 때문입니다. 또한 딥러닝의 "hello world" 튜토리얼이라고 할 수 있습니다.
```
Comment (fuzzythecat) [inline 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md]: 1. 그 이유는 ~~~ 동일한 이미지 크기이며... ~~~의 부분에 다른 말들이 추가되어야 좀 더 자연스러울 것 같습니다
2. 학습 셋 (60,000) -> 학습 셋(60,000) 원문의 오타인 것 같습니다
2. 또한 딥러닝의 "hello world" ... -> 또한을 따라서로 바꾸는 것이 더 자연스러울 것 같습니다

File: 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+[Keras](https://keras.io/)는 인기있고 고수준(high-level)의 딥러닝 API으로 알려져있습니다. 독립적인 오픈 소스 프로젝트일 뿐만 아니라 TensorFlow에 내장되어 있습니다. ```tf.keras``` 를 임포트하여 훌륭한 Keras 프로그램을 작성할 수 있습니다. ```tf.keras``` 를 사용하면 실행 및 ```tf.data```와 같은 기능을 활용할 수 있습니다. 여기서는 기초를 다룰 것 입니다.
+
+저는 몇가지의 딥러닝 용어에 관해 검토할 것입니다. 만약 당신이 딥러닝 입문자라면, 저의 튜토리얼과 저수준(low-level)의 TensorFlow API을 사용한 훨씬 오래된 [MNIST 튜토리얼](https://www.tensorflow.org/tutorials/)을 비교하고 대조해서 보길 권합니다. 얼마나 쉬운 일이 일어났는지 볼 수 있기 때문입니다.  
+
+## Colab에서 실행 (Run this notebook in Colab)
+
+모든 코드는 저의 GitHub에 있습니다. Google의 Colab을 사용하여 [저의 Gitub의 Jupyter Notebook](https://colab.research.google.com/github/margaretmz/deep-learning/blob/master/fashion_mnist_keras.ipynb)을 직접 열어서 실행할 수 있습니다. 빨리 노트북을 열여서 튜토리얼을 따라가고 싶으시다면 이 옵션을 선택해보세요. Colab에 대해 더 알기 원하신다면 [공식 블로그](https://medium.com/tensorflow/colab-an-easy-way-to-learn-and-use-tensorflow-d74d1686e309) 혹은 저의 [블로그](https://medium.com/@margaretmz/running-jupyter-notebook-with-colab-f4a29a9c7156)를 참고하세요.
```
Comment (fuzzythecat) [inline 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md]: _싶으시다면, 원하신다면_ 은 위의 _보길 권합니다_ 와 같은 단어들과 높임 수준에서 차이가 납니다. 하나로 통일해 주는 것이 좋을 것 같습니다

File: 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+# 미리 섞여진 fashoin-mnist 의 학습 데이터와 테스트 데이터 로드
+(x_train, y_train), (x_test, y_test) = tf.keras.datasets.fashion_mnist.load_data()
+
+print("x_train shape:", x_train.shape, "y_train shape:", y_train.shape)
+```
+
+## 데이터 시각화 (Visualize the data)
+
+Jupyter Notebook에서 가장 좋아하는 것은 시각화 입니다. matplotlib 라이브러리의 ```imshow()```를 사용하여 학습 데이터 셋의 이미지를 시각화하여 데이터 셋의 이미지 중 하나를 살펴볼 수 있습니다. 각 이미지는 28x28 모양의 흑백 이미지입니다.
```
Comment (fuzzythecat) [inline 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md]: 제가 Jupyter Notebook에서

File: 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+그런 다음 데이터 크기를 정규화하여 대략적으로 데이터 크기를 동일하게 맞춥니다.
+```
+x_train = x_train.astype('float32') / 255
+x_test = x_test.astype('float32') / 255
+```
+
+## 데이터를 학습/검증/테스트 셋으로 나누기 (Split the data into train/validation/test datasets)
+
+데이터를 임포팅하는 과정에서, 60,000개의 학습 셋과 10,000개의 테스트셋을 얻었다. 이제 학습 셋을 학습 셋/검증 셋으로 나누고자 한다. 딥러닝에서 각 유형의 데이터셋이 사용되는 방법:
```
Comment (fuzzythecat) [inline 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md]: 1. 얻었다 -> 얻을 수 있습니다
2. 나누고자 한다 -> 나누고자 합니다
3. 사용되는 방법 -> 사용되는 방법은 다음과 같습니다

File: 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+## 모델 아키텍처 만들기 (Create the model architecture)
+
+Keras에서 모델을 정의하기위한 두 가지 API:
+
+1. [시퀀스 모델 API](https://keras.io/models/sequential/)(순차 모델 API)
+2. [함수 API](https://keras.io/models/model/)
+
+이 튜토리얼에서는 시퀀스 모델 API를 사용하여  컨볼루션 레이어(CNN)을 반복하는 간단한 CNN 모델을 만들고 풀링 레이어, 드롭 아웃 레이어를 만듭니다. 만약 함수 API에 관심이 있다면 Sara Robinson의 블로그 [Kearas 함수 API와 TensorFlow로 와인 가격 예측하기](https://medium.com/tensorflow/predicting-the-price-of-wine-with-the-keras-functional-api-and-tensorflow-a95d1c2c1b03)를 확인해보세요. <br>
+
+첫 번째 레이어에 입력 데이터 크기(shape)을 정의합니다. 마지막 레이어는 소프트맥스 활성화가 되어있으며, fashion_mnist에서 10가지 범주의 데이터를 분류합니다.
```
Comment (fuzzythecat) [inline 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md]: 1. _입력 데이터의 크기(shape)는 첫 번째 레이어에서만 정의해주면 됩니다_ 가 원문의 느낌을 더 잘 살리는 것 같아요
2. _마지막 레이어의 결과 값은 소프트맥스 활성화 함수를 거치며, 여기서 fashion_mnist의 10가지 범주의 데이터를 분류합니다_ 가 좀 더 자연스러울 것 같아요. 음 원문의 dense layer는 언급하지 않아도 괜찮을까요?

File: 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+# 테스트 셋으로 모델 평가
+score = model.evaluate(x_test, y_test, verbose=0)
+
+# 테스트 정확도
+print('\n', 'Test accuracy:', score[1])
+```
+
+# 예측값 시각화하기 (Visualize the predictions)
+
+이제 훈련 된 모델을 사용하여 테스트 셋을 예측/분류를 하고 ```model.predict(x_test)```   시각화 할 수 있습니다 . 레이블이 빨간색으로 보인다면 실제 레이블과 매칭되지 않음(예측 틀림)을 나타냅니다. 반대로 초록색으로 보인다면 잘 예측한 것 입니다.
```
Comment (fuzzythecat) [inline 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md]: 1. 예측/분류를 -> 예측/분류
2. ```model.predict(x_test)```으로 테스트 셋을 예측/분류하고... 가 더 자연스러울 것 같습니다

File: 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md (hunk 2 above)
Comment (SooDevv) [reply 10_Fashion-MNIST with tf.Keras/10_딥러닝의 Hello World, Fashion-MNIST.md]: 2. 학습 셋(60,000) --> 학습 셋(60,000)은 무슨 뜻인가요??
